Models/CoreBitcoinManager.swift

```diff
@@ -10,22 +10,19 @@ import Foundation
 import BlockchainCertificates
 
 class CoreBitcoinManager : BitcoinManager {
-    func address(for certificate: Certificate, on chain: String) -> String? {
+    func address(for message: String, for signature: String, on chain: String) -> String? {
```

Comment:
Similarly, what's the value of changing this? I'd rather leave this unchanged if we can.

If we can't, then we should have a better method signature `address(for:for:on:)` makes the 1st and 2nd parameters confusing, especially since they're both Strings. And should the second be a new Signature object?